crates/gateway-types/src/reply.rs

```diff
@@ -1838,7 +1838,6 @@ pub mod add_transaction {
     pub struct DeployAccountResponse {
         pub code: String, // TRANSACTION_RECEIVED
         pub transaction_hash: TransactionHash,
-        pub address: ContractAddress,
```

Comment:
Should probablu remove `deny_unknown_fields`. We should also confirm that the other delcare, deploy responses have also not changed/